pkg/chain/operator_id.go

```diff
@@ -0,0 +1,15 @@
+package chain
+
+import "math"
+
+// OperatorID is a unique identifier of an operator assigned by the Sortition
+// Pool when the operator enters the pool for the first time. ID is never
+// changing for the given operator address.
+type OperatorID = uint32
```

Comment:
Is there any particular reason to use a type alias here? The default is rather type definition (`type OperatorID uint32`). See https://yourbasic.org/golang/type-alias/

Reply:
I think it's better to leave it as is. Using an alias requires less explicit conversions in the code.